### PR TITLE
[14.0][ADD]Added Module purchase_order_line_menu

### PR DESCRIPTION
**Pull request by Freni-OSI**

### CODE REVIEW

**Review by dreispt — COMMENTED** [review]
Looks great!

**Review by dreispt — APPROVED** [review]

File: purchase_order_line_menu/views/purchase_order_line_views.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <field name="arch" type="xml">
+            <pivot string="Order Lines" sample="1">
+                <field name="product_id" type="row" />
+                <field name="product_qty" type="col" />
+                <field name="price_subtotal" type="measure" />
+            </pivot>
+        </field>
+    </record>
+    <record id="action_purchase_orders_lines" model="ir.actions.act_window">
+        <field name="name">Order Lines</field>
```
Comment (bodedra) [inline purchase_order_line_menu/views/purchase_order_line_views.xml]:
```suggestion
        <field name="name">Purchase Order Lines</field>
```

File: purchase_order_line_menu/views/purchase_order_line_views.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">purchase.order.line</field>
+        <field name="view_mode">tree,pivot</field>
+    </record>
+
+    <menuitem
+        id="purchase_order_line_menu"
+        name="Order Lines"
```
Comment (bodedra) [inline purchase_order_line_menu/views/purchase_order_line_views.xml]:
```suggestion
        name="Purchase Order Lines"
```

File: purchase_order_line_menu/views/purchase_order_line_views.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                <field name="product_qty" type="col" />
+                <field name="price_subtotal" type="measure" />
+            </pivot>
+        </field>
+    </record>
+    <record id="action_purchase_orders_lines" model="ir.actions.act_window">
+        <field name="name">Order Lines</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">purchase.order.line</field>
+        <field name="view_mode">tree,pivot</field>
```
Comment (bodedra) [inline purchase_order_line_menu/views/purchase_order_line_views.xml:17]:
```suggestion
        <field name="view_mode">tree,form</field>
```

**Review by bodedra — COMMENTED** [review]
LG :+1: 

Added minor suggestions.

File: purchase_order_line_menu/views/purchase_order_line_views.xml (hunk 3 above)
Comment (dreispt) [reply purchase_order_line_menu/views/purchase_order_line_views.xml:17]: No, we want pivot.

File: purchase_order_line_menu/views/purchase_order_line_views.xml (hunk 1 above)
Comment (dreispt) [reply purchase_order_line_menu/views/purchase_order_line_views.xml]: Yes

**Review by bodedra — APPROVED** [review]
Functional reviewed. :+1:

**Comment by HviorForgeFlow** [comment]
This is not meant to be a blocking comment.
Wouldn't fit better on Purchase Report Menu? and maybe calling this module purchase_order_line_report?
Just guessing

**Pull request merged**